app/templates/Articles/add.php

```diff
@@ -0,0 +1,12 @@
+<!-- File: templates/Articles/add.php -->
```

Comment:
It's a HTML comment, please refer https://developer.mozilla.org/en-US/docs/Web/API/Comment

> The Comment interface represents textual notations within markup; although it is generally not visually shown, such comments are available to be read in the source view.

It's viewable from the source view. Is it OK to expose the comment to the end user? Please kindly clarify.

Reply:
Since it's copied from the CakePHP tutorial, it's OK for now. No need to remove this line and others too, but I hope it's not practice in the actual project.

Reply:
Additionally, I would like to share new lesson for you. It's not mentioned within the CakePHP tutorial. 

Please refer https://book.cakephp.org/bake/2/en/usage.html#code-generation-with-bake

CakePHP has `bake` command (I think you notice already) that able to generate / bake many things (following it's standard).

Please try to run the bake command to generate a template file. Then please compare the code copied from the tutorial and the generated one. Please kindly share what's the difference between both.

Reply:
on the users and tags I use the bake commands to generate the templates.

Using commands we have more code, code block comments and html table format

Reply:
> Using commands we have more code, code block comments and html table format

Thank you for the explanation. In the future, it's better to follow the CakePHP standard on generating the respective files from bake command.

_______________________

Back to my original question https://github.com/ArielGalvez/tutorial-cakephp/pull/2#discussion_r1377413333

Using HTML comment can be visible to the end user through view source. It'll be NG if we put the unnecessary comment, in this case the file path will be visible.

Reply:
Sure I can examplain better:
The comment wont be visible in the user interface. But the comment will be visible if user inspect the html code, so in real proyects is bad idea adding commets like this:
`<!-- comment -->`
Also if a comment is needed we can use php comments like this:
```
// line comment
or
/* 
  * block comment
  */
```